templates/repo/issue/fields/checkboxes.tmpl

```diff
@@ -0,0 +1,12 @@
+<div class="field">
+	{{template "repo/issue/fields/header" .}}
+	{{$field := .}}
+	{{range $i, $opt := .Attributes.options}}
+		<div class="field">
+			<div class="ui checkbox {{if $opt.required}}read-only{{end}}">
+				<input type="checkbox" name="form-field-{{$field.ID}}-{{$i}}" {{if $opt.required}}checked=""{{end}}>
```

Comment:
`{{if $opt.required}}checked=""{{end}}`

Is it correct? required + checked?

Reply:
Yes, you are right. Sorry, I'm really not good at HTML.😂